### DISCUSSION TITLE
How long will the battery keep the simulated RTC running

### DISCUSSION THREAD

**Discussion by Prhewett** [PiSupply/PiJuice#704] [Ideas]
I am building a solution that has a need to power down within a few minutes of supply power going off, and then power up when supply power is restored and at 0%. 
What I also need in this project is an RTC to keep the Pi on a time close to UTC. In my application the supply power may be off for between 1 and 36 hours.
I see there is a simulated RTC in the PiJuice hat. 
Can anyone give me any guidance as to the likely max time the PiJuice battery will keep the simulated RTC ticking over ?

**Comment by tvoverbeek** [comment]
It uses very little power. I've seen the RTC still running on one of my PiJuices when the power off was more than 2 weeks. So 36 hours would not be a problem (provided there is still some charge in the battery)
